spec/spec_helper.rb

```diff
@@ -1,7 +1,7 @@
 require 'simplecov'
 require 'simplecov-rcov'
 SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
-SimpleCov.minimum_coverage 100
```

Comment:
I know, just saying that the earlier we get to 100% the better. The effort to get up to 100% now is going to be less than if we leave it at 90% for months